docs/configure/overview.md

```diff
@@ -54,7 +54,7 @@ Additionally, you can also provide additional feature flags to your Storybook co
 | Configuration element | Description                                                                                                                                                       |
 | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `storyStoreV7`        | Configures Storybook to load stories [on demand](#on-demand-story-loading), rather than during boot up <br/> `features: { storyStoreV7: true }`                   |
-| `buildStoriesJson`    | Generates a `stories.json` file to help story loading with the on-demand mode <br/> `features: { buildStoriesJson: true }`                                        |
+| `buildStoriesJson`    | Generates an `index.json` (formerly `stories.json`) file to help story loading with the on-demand mode <br/> `features: { buildStoriesJson: true }`               |
```

Comment:
I think this change is good, but I'll give a bit more context about `buildStoriesJson`:
1. You don't have to set this flag at all if `storyStoreV7` is set to true, because that does the same. `storyStoreV7` is set to true by default, even if you don't have it in main.js.
2. You don't have to set this flag at all in Storybook 7 because `storyStoreV7` is true by default
3. `buildStoriesJson` now produces both a stories.json and an index.json file